### PR TITLE
fix(notification-actions): disable add action dropdown if no notification actions to add

### PR DESCRIPTION
**Pull request by cathteng**
Before (weird tiny dropdown)
Screenshot 2023-12-06 at 15 32 12

After
No notification actions to add
Screenshot 2023-12-06 at 15 28 42

No permission to add (regardless if there are any notification actions available to add)
Screenshot 2023-12-06 at 15 16 21

For ER-1765

### CODE REVIEW

File: static/app/components/notificationActions/notificationActionManager.tsx [hunk 1]
```diff
@@ -213,26 +213,37 @@ function NotificationActionManager({
...
   };
 
+  const menuItems = getMenuItems();
+
+  const toolTipText = () => {
+    if (disabled) {
+      return t('You do not have permission to add notification actions for this project');
+    }
+    if (menuItems.length === 0) {
+      return t('You do not any notification actions to add');
```
Comment (sentaur-athena) [inline static/app/components/notificationActions/notificationActionManager.tsx:223]: You do not have?

Comment (cathteng) [reply static/app/components/notificationActions/notificationActionManager.tsx:223]: oops ty

File: static/app/components/notificationActions/notificationActionManager.tsx [hunk 2]
```diff
@@ -213,26 +213,37 @@ function NotificationActionManager({
     return menuItems;
   };
 
+  const menuItems = getMenuItems();
```
[the remark this answers is not in the document]
Comment (cathteng) [reply static/app/components/notificationActions/notificationActionManager.tsx:216]: sure

File: static/app/components/notificationActions/notificationActionManager.tsx [hunk 3]
```diff
@@ -210,29 +210,38 @@ function NotificationActionManager({
         },
       });
     });
-    return menuItems;
+    return dropdownMenuItems;
+  }, [actionsMap, availableServices, notificationActions, project, updateAlertCount]);
+
+  const toolTipText = () => {
```
Comment (nhsiehgit) [inline static/app/components/notificationActions/notificationActionManager.tsx:216]: This doesn't need to be a function either right?

**Review by nhsiehgit — APPROVED** [review]

**Pull request merged**